### DISCUSSION TITLE
AlpineJS Framework - Build with AlpineJS (TailwindCSS + Webpack integrated)

### DISCUSSION THREAD

**Discussion by johannesronaldsson** [alpinejs/alpine#2776] [2. Show and tell]
# AlpineJS Framework

A simple framework that is build with AlpineJS, create powerful web applications with the simplicity of AlpineJS. 

**Last Update: 2022-03-24 01:38 UTC**

Check out the repo I've created with the latest version. Enjoy and ideas are most welcome!
https://github.com/getneurotic/alpinejs-framework

**Comment by Tim-Wils** [comment]
It's always nice to see different kinds of AlpineJS approaches everyday :)
Nice job!

**Reply by johannesronaldsson** [reply]
Thanks, it's an ongoing fun-project, so will update and share repo once it has more shape. I'm just very very happy that AlpineJS exists! :D
